bigbluebutton-html5/imports/ui/components/video-provider/video-dock/component.jsx

```diff
@@ -74,6 +72,16 @@ class VideoDock extends Component {
     }, 0);
   }
 
+  getUsersWithActiveStreams() {
```

Comment:
Like stated before, why you dont filter the user-list on the container itself instead of the component? You have all data needed there.